docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -2457,6 +2458,34 @@ array_replace_all(array, from, to)
 
 - list_replace_all
 
+### `array_reverse`
+
+Returns the array with the order of the elements reversed.
+
+```
+array_reverse(array)
+```
+
+#### Arguments
+
+- **array**: Array expression.
+  Can be a constant, column, or function, and any combination of array operators.
+
+#### Example
+
+```
+❯ select array_reverse([1, 2, 3, 4]);
++------------------------------------------------------------+
+| array_reverse(List([1, 2, 3, 4]))                          |
++------------------------------------------------------------+
+| [4, 3, 2, 1]                                               |
++------------------------------------------------------------+
+```
+
+#### Aliases
+
+- list_reverse
```

Comment:
I think also need to add doc stub for list_reverse?

e.g.

https://github.com/apache/arrow-datafusion/blob/7a5f2054305fd92852b589473afbc9bb034379d7/docs/source/user-guide/sql/scalar_functions.md?plain=1#L2718-L2720